src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -32,6 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: J. Henrichs, Bureau of Meteorology
+# Modified: R. W. Ford, STFC Daresbury Lab
 
```

Comment:
The only changes to this file appear to be your name and a blank line. If that's so, please could you revert.

Reply:
Oops, reverted.